MainCode.java

```diff
@@ -1,7 +1,13 @@
-
+import java.util.Scanner;
 public class MainCode {
 	public static void main(String[] args) {
+		Scanner keyboardInput = new Scanner(System.in);
+		
 		System.out.println("Hello! Want to play the game?");
 		System.out.println("How are you?");
+		String userInput = keyboardInput.nextLine();
+		if(userInput == "good".toLowerCase()) {
```

Comment:
This only changes "good" to lowercase for the comparison, not the user input. They can still enter "Good" and get nothing in return